projects/control-service/projects/pipelines_control_service/src/integration-test/java/com/vmware/taurus/graphql/it/GraphQLExecutionsLogsUrlIT.java

```diff
@@ -132,7 +133,7 @@ public void testExecutions_filterByStartTimeGte() throws Exception {
                     "{"
                         + "\"filter\": {"
                         + "      \"startTimeGte\": \""
-                        + dataJobExecution3.getStartTime()
+                        + getTimeAccurateToMicroSecond()
```

Comment:
shouldn't you use a method that accepts dataJobExecution3.getStartTime()?

Reply:
strange let me look into this.